index.html

```diff
@@ -185,15 +302,16 @@ <h1 class="greet">
           <li class="angelist"><a href="#"></a><img src="assets/Vector.png"></li>    
         </ul>
         <div class="resume">
-          <button class="button" id="resume">Get my resume</button>
+          <button type="button" class="button" id="resume">Get my resume</button>
         </div>
-
-        
+      </div>
+        <div class="grid_item2">
           <ul class="skills">
-              <ul class="languages">  
+              <ul class="languages"> 
                 <li><p>Languages</p></li>
-                <li><img src="assets/Unionlangs.png"></li>
+                <li class="arrow"><img src="assets/Unionlangs.png"></li>
```

Comment:
[OPTIONAL] You are almost there. Kindly try to fix it.
 
| Your Design   | FIGMA Design  | Solution  |
| ------------- | ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/213889423-ffaa6bfb-2f14-4978-a443-73fd822f5a40.png)  | ![image](https://user-images.githubusercontent.com/61361037/213889406-484f6ec7-8373-4923-8a62-e058aca7d6fe.png)  | ![image](https://user-images.githubusercontent.com/61361037/213889394-11b84b93-e82e-46f7-b709-c6f22a731a56.png) |